pkg/client/registry.go

```diff
@@ -6,6 +6,27 @@ import (
 	clusterC "github.com/spectrocloud/hapi/spectrocluster/client/v1"
 )
 
+func (h *V1Client) GetRegistryPackByName(registryName string) (*models.V1PackRegistry, error) {
+	client, err := h.getClusterClient()
+	if err != nil {
+		return nil, err
+	}
+
+	params := clusterC.NewV1RegistriesPackListParams()
+	registries, err := client.V1RegistriesPackList(params)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, registry := range registries.Payload.Items {
+		if registry.Metadata.Name == registryName {
+			return registry, nil
+		}
+	}
+
+	return nil, fmt.Errorf("Registry '%s' not found.", registryName)
+}
+
 func (h *V1Client) GetRegistryHelmByName(registryName string) (*models.V1HelmRegistry, error) {
```

Comment:
Please change method name to `GetHelmRegistryByName`

Reply:
Done